helm-chart/values.yaml

```diff
@@ -164,16 +164,109 @@ monitor:
 
 secrets:
   localSettings: 
-    ref: # Reference to an existing secret in the cluster
-    data: |-
-    # This is where the local settings go if you're on your local. Please ensure indentation for the whole file is consistent with the first line indentation
+    ref: 
+    # Add the value of these variables in the data: field for local use only. 
+    # Also, it is understood that if you're adding one value you're adding variable values 
+    DATABASES_HOST:
```

Comment:
Why do we have a data element ? Why not just go DATABASE_HOST: test

Reply:
Earlier it was to differentiate between certain secrets that were individually defined. I stuck with the same convention for easy readability by devs in future.

Reply:
Except that this isn't a standard convention - I have not seen other helm charts use this.

Reply:
Noted. I'll remove the `data:` element.